src/engine/sys/sdl_glimp.cpp

```diff
@@ -1864,10 +1865,21 @@ static void GLimp_InitExtensions()
 
 	// Texture - others
 	glConfig2.textureAnisotropyAvailable = false;
-	if ( LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, EXT_texture_filter_anisotropic, r_ext_texture_filter_anisotropic->value ) )
+	if ( LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, EXT_texture_filter_anisotropic, r_ext_texture_filter_anisotropic.Get() > 0 ) )
 	{
 		glGetFloatv( GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT, &glConfig2.maxTextureAnisotropy );
 		glConfig2.textureAnisotropyAvailable = true;
+
+		// Bound texture anisotropy.
+		if ( r_ext_texture_filter_anisotropic.Get() > glConfig2.maxTextureAnisotropy )
+		{
+			// HACK: We want to set the current value, not the latched value
```

Comment:
I don't think we should be setting the value anyway (I see that the existing code does this). If the driver lacks an extension, we don't set the cvar to 0 but rather record which value should be used in glConfig(2). Analogously, we could store the effective max anisotropy there instead of the cvar.

Reply:
Right, we don't need to se the cvar, it was a hack for reading the value in `tr_image`, we could just use some glconfig `int` instead.

but we may still have to implement the `Cvar::Unlatch` function the day we port that other code:

```c++
// HACK: We want to set the current value, not the latched value
Cvar::ClearFlags("r_customwidth", CVAR_LATCH);
Cvar::ClearFlags("r_customheight", CVAR_LATCH);
Cvar_Set( "r_customwidth", va("%d", glConfig.vidWidth ) );
Cvar_Set( "r_customheight", va("%d", glConfig.vidHeight ) );
Cvar::AddFlags("r_customwidth", CVAR_LATCH);
Cvar::AddFlags("r_customheight", CVAR_LATCH);
```

Reply:
Done.